src/Core/x/xstransvc.cpp

```diff
@@ -140,7 +133,7 @@ int8* xST_xAssetID_HIPFullPath(uint32 aid)
 // func_8004BE60
 void XST_reset_raw()
 {
-    memset(&g_xstdata,0,0x1144);
+    memset(&g_xstdata, 0, 0x1144);
```

Comment:
see if 0x1144 can be  replaced with `sizeof(g_xstdata)`. If that is what that number is, add it really quick and I'll merge. Otherwise, I'll merge it now

Reply:
Or `sizeof(st_STRAN_DATA)` rather